README.md

```diff
@@ -17,4 +17,13 @@ observerName = "observer"
 
 prefixes = (',', '@')
 ```
-Switch the values for your own server's IDs and names.
\ No newline at end of file
+Switch the values for your own server's IDs and names.
+
+## Building and running using docker
+To build the docker image, run the following command in the root directory
+
+```docker build -t botc .```
+
+When the image is built, you can run the bot using the following command
+
+```docker run -v .:/app -v $(pwd)/bot_configs/bot-name.py:/app/config.py -d --name bot-name botc```
```

Comment:
Can you replace both instances with `bot-name` with `${BOT_NAME}` to make it clear that it is a variable to be filled in by the caller rather than something that can just be copy/pasted blindly?

Also, maybe add a line above it for
  `BOT_NAME=<George|Leo|Quinn|Tiptoe>` to also make it clear what values should be used and to encourage setting a temp ENV variable so you can _then_ copy/paste